libs/api/domains/education/src/lib/graphql/main.resolver.ts

```diff
@@ -38,18 +33,33 @@ export class MainResolver {
     @Args('input', { type: () => FetchEducationSignedLicenseUrlInput })
     input: FetchEducationSignedLicenseUrlInput,
   ): Promise<SignedLicense> {
-    const responseStream = await this.educationService.downloadPdfLicense(
+    const url = await this.educationService.downloadPdfLicense(
       user.nationalId,
       input.licenseId,
     )
-
-    const url = await this.s3Service.uploadFileFromStream(responseStream, {
-      fileName: uuid(),
-      bucket: this.config.fileDownloadBucket,
-    })
     if (url === null) {
       throw new ApolloError('Could not create a download link')
     }
     return { url }
   }
+
+  @Query(() => [ExamOverview])
+  educationExamOverviews(@CurrentUser() user: User): Promise<ExamOverview[]> {
```

Comment:
I believe the naming should incline that the user's family member's exam overviews are also fetched.

Otherwise agree that the family handling should be done in the service :+1:

Reply:
Then it's the same for the name of the query.

Reply:
Not sure I understand, are you saying we should use the naming `ExamFamilyOverview` instead of `ExamOverview`?

Reply:
Yeah something like that.

Reply:
Both for the query name and the service method name